config/remotePlugin.js

```diff
@@ -349,6 +349,32 @@ const routeExtensions = [
       required: ['SIGNUP'],
     },
   },
+  {
+    type: 'console.page/route',
+    properties: {
+      path: '/stonesoup',
+      exact: false,
+      component: {
+        $codeRef: 'NotFound',
+      },
+    },
+    flags: {
+      required: ['SIGNUP'],
+    },
+  },
```

Comment:
We don't need to add the deprecated `console` extensions anymore.

Reply:
Shall I remove the others?

Reply:
@jeff-phillips-18 may as well if they are redundant and unnecessary.